From ecffd17b443b355f6042fde40e40c3d1a754944b Mon Sep 17 00:00:00 2001
From: Andy CrossGate Yan <GeForce8800Ultra@gmail.com>
Date: Sat, 24 Sep 2022 07:44:20 +0000
Subject: [PATCH 08/10] Skip mounting /my_manifest on GSIs

On OPlus devices, /my_manifest/build.prop unnecessarily overrides a ton of props
All my_* partitions are supposedly useless on GSIs, but only skip this for now

Change-Id: I638fd8ed2cfd7ca835a8e46fd235ebf08c2460a4
---
 target/product/gsi/gsi_skip_mount.cfg | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/target/product/gsi/gsi_skip_mount.cfg b/target/product/gsi/gsi_skip_mount.cfg
index 28f4349666..612c3bc814 100644
--- a/target/product/gsi/gsi_skip_mount.cfg
+++ b/target/product/gsi/gsi_skip_mount.cfg
@@ -7,3 +7,6 @@
 /product/*
 /system_ext/*
 /system/*
+# Skip OPlus "my_manifest" mountpoints.
+/mnt/vendor/my_manifest
+/my_manifest
-- 
2.25.1

